libbeat/autodiscover/autodiscover.go

```diff
@@ -218,7 +215,7 @@ func (a *Autodiscover) handleStart(event bus.Event) bool {
 		dynFields := a.meta.Store(hash, meta)
 
 		if a.configs[eventID][hash] != nil {
-			a.logger.Debugf("Config %v is already running", config)
+			a.logger.Debugf("Config %v is already running", common.DebugString(config, true))
```

Comment:
What would happen without this? I'm a bit concerned that at some point we add a new debug message and forget to do this

Reply:
All these debug message print configuration. I was able to spot this:
`2020-04-28T11:22:21.892Z	DEBUG	[autodiscover]	autodiscover/autodiscover.go:199	Generated config: map[hosts:[172.17.0.2:6379] metricsets:[info keyspace] module:redis password:passpass]`

This one was due to this debug logging: https://github.com/elastic/beats/blob/09fd4df169324d4e02b2f3bf3c074d7501f21968/libbeat/autodiscover/autodiscover.go#L197

In this I changed all the similar ones in this method.